.rubocop.yml

```diff
@@ -7,6 +7,7 @@ AllCops:
     - 'bin/**/*'
     - 'config/initializers/devise.rb'
     - 'db/migrate/20160405212342_initial_schema.rb'
+  TargetRubyVersion: 2.3
```

Comment:
this seemed to give travis some fits last time I tried it, so ㊗️ for getting it to pass.